nextstep-mvc/src/main/java/nextstep/mvc/tobe/JsonView.java

```diff
@@ -25,9 +25,7 @@ public void render(Map<String, ?> model, HttpServletRequest request, HttpServlet
         String json = JsonUtils.toJson(getValues(model));
         log.debug("json data: {}", json);
 
-        writer.write(json);
-        writer.flush();
-        writer.close();
+        JsonUtils.getInstance().writeValue(writer, model);
```

Comment:
JsonUtils 클래스가 외부에서 사용할 수 있는 `writeValue()` 메소드를 제공하면 어색함이 줄어들지 않을까요? 유틸성 클래스이기 때문에 인스턴스를 제공하는 것 보다 메소드를 제공하는 것이 더 좋을 것 같아요.
그리고 필요없는 25-26 line은 제거해도 될 것 같구요.
```suggestion
        JsonUtils.writeValue(writer, getValues(model));
```